recipe/meta.yaml

```diff
@@ -36,3 +36,6 @@ extra:
     - SylvainCorlay
     - wolfv
     - marimeireles
+  additional-platforms:
```

Comment:
I wonder how this works here.
The `additional-platforms` thingy is the BioConda way. The Conda-forge way is https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6726/files and https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6726/files

Reply:
Please ignore! I was commenting on old code.

Reply:
It does not, I mistakenly put `additional-platforms` there. My BioConda recipe (https://github.com/bioconda/bioconda-recipes/pull/52250) depends on indicators. I thought BioConda and conda-forge add ARM support the same way. I removed this in a later commit.